RELEASE-NOTES.txt

```diff
@@ -3,6 +3,10 @@ Unreleased
 * [*] Tablet view fixes for inserter button. https://github.com/wordpress-mobile/gutenberg-mobile/pull/3602
 * [**] Fixed an issue where pressing enter inside a text-based block was not creating a new block when using Gboard [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3590]
 
+1.55.1
+------
+* [*] Improve unsupported block message for reusable block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3621]
+
```

Comment:
I'm not sure if we should add a line for
`[*] Fix: RNMobile borderRadius value setting [https://github.com/WordPress/gutenberg/pull/32717]`

Reply:
Yeah I think so, I'll add it to the release notes, thanks for the heads up 🙇 !